disputer/disputer.js

```diff
@@ -142,9 +142,10 @@ class Disputer {
         continue;
       }
 
+      const gasLimit = (await dispute.estimateGas({ from: this.account })) * 1.5;
```

Comment:
We need to either catch `estimateGas()` call or refactor and add it to the same try-catch as the `send()` call, because `estimateGas()` will throw if `dispute()` would throw normally

Reply:
Is this true? if the `dispute()` would have failed then the call would have thrown and been caught a few lines up in the `try/catch` for the `.call`.

In either case, I moved it upwards to the preview try-catch block.

PTAL.

Reply:
Yeah I think if you move it into the same try-catch as the `.call()` then you're fine. You do need to catch it though: https://web3js.readthedocs.io/en/v1.2.6/web3-eth-contract.html#methods-mymethod-estimategas